docs/v4-upgrade.md

```diff
@@ -87,8 +94,24 @@ nodeModulesLoader.exclude.push(/some-library/) // replace `some-library` with
                                                // the actual path to exclude
 ```
 
-### Example upgrades
+## Namespaces
+
+The compiled packs in the public directory are now stored under namespaces:
+
+- JavaScripts are stored under `js`
+- Stylesheets are stored under `css`
+- Other resources are stored under `media`
+
+```rb
+# Before
+"runtime~hello_react" => "/packs/runtime~hello_react-da2baf7fd07b0e8b6d17.js"
+
+# After
+"runtime~hello_react" => "/packs/js/runtime~hello_react-da2baf7fd07b0e8b6d17.js"
+```
+
+## Example Upgrades
 
 This is what an upgrade to Webpacker 4 looked like for existing Rails apps (please contribute yours!):
 
-- https://github.com/connorshea/ContinueFromCheckpoint/pull/77
```

Comment:
Why remove the list syntax here?

Reply:
I did a few markup changes based on the MD linter in VS Code. Not sure how I removed the list syntax, but I've corrected it 🙂